app/views/command-line.html

```diff
@@ -0,0 +1,77 @@
+                      <img src="images/openshift-logo.svg" />
+                    </div>
+                    <div class="col-md-9">
+                      <h1>OpenShift by Red Hat&reg;</h1>
+                      <h2 id="cli">Command Line Tools</h2>
+                      <p>
+                        With the OpenShift command line interface (CLI), you can create applications and manage OpenShift projects from a terminal.
+                        <span ng-if="cliDownloadURLPresent">
+                          You can download the <code>oc</code> client tool using the links below. For more information about downloading and installing it, please refer to the <a href="{{'get_started_cli' | helpLink}}">Get Started with the CLI</a> documentation.
+                        </span>
+                        <span ng-if="!cliDownloadURLPresent">
+                          Refer to the <a href="{{'get_started_cli' | helpLink}}">Get Started with the CLI</a> documentation for instructions about downloading and installing the <code>oc</code> client tool.
+                        </span>
+                        <div ng-if="cliDownloadURLPresent">
+                          <label class="cli-download-label">Download <code>oc</code>:</label>
+                          <div ng-repeat="(key, value) in cliDownloadURL">
+                            <a href="{{value}}" class="cli-download-link">
+                              {{key}}
+                              <i class="fa fa-external-link"></i>
+                            </a>
+                          </div>
+                        </div>
+                      </p>
+                      <p>
+                        After downloading and installing it, you can start by logging in using<span ng-if="sessionToken"> this current session token</span>:
+                        <div class="code prettyprint ng-binding" ng-if="sessionToken">
+                          oc login {{loginBaseURL}} --token=<span ng-show="showSessionToken">{{sessionToken}}</span><a href="#" ng-click="showSessionToken = !showSessionToken" ng-show="!showSessionToken">...click to show token...</a>
+                        </div>
+                        <pre class="code prettyprint ng-binding" ng-if="!sessionToken">
+        oc login {{loginBaseURL}}
+        </pre>
+                      </p>
+                      <p>
+                        <span class="pficon pficon-warning-triangle-o" style="cursor: help; vertical-align: middle;"></span>
+                        <strong>It is strongly advised not to share your API token with anyone.</strong>
+                      </p>
+
+                      <p>After you login to your account you will get list of existing projects between which you can switch:
+                         <pre class="code prettyprint">oc project <i>project-name</i></pre>
+                      </p>
+
+                      <p>In case you don't have any existing project, you can create one:
```

Comment:
Although, they may not even be able to create one if the admin has restricted it... @spadgett what do you think about this one